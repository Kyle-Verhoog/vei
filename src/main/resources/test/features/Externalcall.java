public class Externalcall {
  public Externalcall() {}
  public int m(java.util.Vector x) {
    return x.size();
  }
}