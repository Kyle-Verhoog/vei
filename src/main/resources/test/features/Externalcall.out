PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
INT ┃ int
IDENTIFIER ┃ m
( ┃ (
IDENTIFIER ┃ java
. ┃ .
IDENTIFIER ┃ util
. ┃ .
IDENTIFIER ┃ Vector
IDENTIFIER ┃ x
) ┃ )
{ ┃ {
RETURN ┃ return
IDENTIFIER ┃ x
. ┃ .
IDENTIFIER ┃ size
( ┃ (
) ┃ )
; ┃ ;
} ┃ }
} ┃ }